polkadot/node/subsystem-bench/src/lib/statement/test_state.rs

```diff
@@ -130,6 +131,8 @@ impl TestState {
 				let mut receipt = receipt_templates[candidate_index].clone();
 				receipt.descriptor.set_para_id(Id::new(core_idx as u32 + 1));
 				receipt.descriptor.set_relay_parent(block_info.hash);
+				receipt.descriptor.set_core_index(CoreIndex(core_idx as u32));
+				receipt.descriptor.set_session_index(0);
```

Comment:
we should have a constant somwehere for the session index. I see it's being used in a couple of places here

Reply:
we don't. We calculate it every time using NODE_UNDER_TEST constant
```
let own_backing_group_index = groups
	.iter()
	.position(|group| group.iter().any(|v| v.0 == NODE_UNDER_TEST))
	.unwrap();
```